README.md

```diff
@@ -146,126 +138,23 @@ Clone the repository
 git clone git@github.com:OpenZeppelin/cairo-contracts.git
```

Comment:
I cannot select it, but for consistency, the "Clone the repository" line above should end in a dot.